### DISCUSSION TITLE
打包的时候一直提示下载文件electron-v31.7.1-win32-x64.zip，导致打包不成功

### DISCUSSION THREAD

**Discussion by superkamiu** [alex8088/electron-vite#642] [Q&A]
我在使用npm run build:win打包的时候， 一直提示：downloading     url=https://github.com/electron/electron/releases/download/v31.7.1/electron-v31.7.1-win32-x64.zip size=111 MB parts=8，我已将此压缩包下载到本地的路径：C:\Users\Administrator\AppData\Local\electron-builder\Cache，但是等待一段时间后，返回如下错误：
• electron-builder  version=24.13.3 os=10.0.19045
  • loaded configuration  file=C:\Users\Administrator\Desktop\canyin4\electron-builder.yml
  • writing effective config  file=dist\builder-effective-config.yaml
  • skipped dependencies rebuild  reason=npmRebuild is set to false
  • packaging       platform=win32 arch=x64 electron=31.7.1 appOutDir=dist\win-unpacked
  • downloading     url=https://github.com/electron/electron/releases/download/v31.7.1/electron-v31.7.1-win32-x64.zip size=111 MB parts=8
  • retrying        attempt=1
  • retrying        attempt=2
  • retrying        attempt=3
  ⨯ part download request failed with status code 401
github.com/develar/app-builder/pkg/download.(*Part).doRequest
        /Volumes/data/Documents/app-builder/pkg/download/Part.go:126
github.com/develar/app-builder/pkg/download.(*Part).download
        /Volumes/data/Documents/app-builder/pkg/download/Part.go:67
github.com/develar/app-builder/pkg/download.(*Downloader).DownloadResolved.func1.1
        /Volumes/data/Documents/app-builder/pkg/download/downloader.go:155
github.com/develar/app-builder/pkg/util.MapAsyncConcurrency.func2
        /Volumes/data/Documents/app-builder/pkg/util/async.go:68
runtime.goexit
        /usr/local/Cellar/go/1.17/libexec/src/runtime/asm_amd64.s:1581  
  ⨯ C:\Users\Administrator\Desktop\canyin4\node_modules\.store\app-builder-bin@4.0.0\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Exit code:
1  failedTask=build stackTrace=Error: C:\Users\Administrator\Desktop\canyin4\node_modules\.store\app-builder-bin@4.0.0\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Exit code:
1
    at ChildProcess.<anonymous> (C:\Users\Administrator\Desktop\canyin4\node_modules\.store\builder-util@24.13.1\node_modules\builder-util\src\util.ts:252:14)
    at Object.onceWrapper (node:events:634:26)
    at ChildProcess.emit (node:events:519:28)
    at ChildProcess.cp.emit (C:\Users\Administrator\Desktop\canyin4\node_modules\.store\cross-spawn@7.0.3\node_modules\cross-spawn\lib\enoent.js:34:29)
    at maybeClose (node:internal/child_process:1105:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:305:5)

**Comment by alex8088** [comment]
网络问题，脚手架创建时勾选代理选项会生成相关下载配置(npmrc)。
